<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeForm</class>
 <widget class="QWidget" name="WelcomeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="logo_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="welcomeToLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome to </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="chooseLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose your position to begin</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="landlordButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Landlord</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tenantButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Tenant</string>
   </property>
  </widget>
  <widget class="QToolButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Already have an account? Log in here</string>
   </property>
  </widget>
  <widget class="QWidget" name="Welcome" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>590</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="micromanage_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Welcome</zorder>
  <zorder>logo_label</zorder>
  <zorder>welcomeToLabel</zorder>
  <zorder>chooseLabel</zorder>
  <zorder>landlordButton</zorder>
  <zorder>tenantButton</zorder>
  <zorder>loginButton</zorder>
  <zorder>micromanage_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
